acme/client.go

```diff
@@ -432,13 +434,9 @@ func (c *Client) chooseSolvers(auth authorization, domain string) map[int]solver
 				logf("[INFO][%s] acme: Could not find solver for: %s", domain, auth.Challenges[idx].Type)
 			}
 		}
-
-		// If we can solve the whole combination, return the solvers
-		if len(solvers) == len(combination) {
-			return solvers
-		}
 	}
-	return nil
+
+	return solvers
```

Comment:
I'm not sure I understand the changes here. What exactly is the current behavior and what is the new behavior?